samples/unit-tests/legend/legend-general/demo.js

```diff
@@ -130,6 +130,18 @@ QUnit.test('Legend redraws', function (assert) {
     chart.series[0].hide(true);
 
     assert.strictEqual(visible, false, 'Legend item text has changed (#2165)');
+
+    chart.series[0].remove();
+
+    chart.update({
+        legend: {
+            title: {
+                text: 'Legend'
+            }
+        }
+    });
+
+    assert.strictEqual(chart.legend.group.visibility === 'hidden', true, 'Legend title is not visible (#17167)');
```

Comment:
Instead of pulling the property value from an object, it's safer to use the `attr` method:
`chart.legend.group.attr('visibility')`